internal/build/docker_test.go

```diff
@@ -193,6 +213,46 @@ type pathContent struct {
 	contents string
 }
 
+func (f *testFixture) assertFileInImage(tag digest.Digest, path string) {
```

Comment:
no need to make a separate func for this, just assert the file exists with contents `""`

Reply:
...then my lazy ass might prefer that you run the command `echo hello world > foo` so you can still use the existing func? 🤷‍♀️